parcel.go

```diff
@@ -34,27 +54,68 @@ func (s ParcelStore) GetByClient(client int) ([]Parcel, error) {
 	// здесь из таблицы может вернуться несколько строк
 
 	// заполните срез Parcel данными из таблицы
+	rows, err := s.db.Query("SELECT number, client, status, address, created_at FROM parcel WHERE client = ?",
+		client)
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
 	var res []Parcel
 
+	for rows.Next() {
+		p := Parcel{}
+
+		err := rows.Scan(&p.Number, &p.Client, &p.Status, &p.Address, &p.CreatedAt)
+		if err != nil {
+			return res, err
```

Comment:
Лучше возвращать пустоту если ошибка
```suggestion
			return nil, err
```

Reply:
Добрый день! Исправил.